benchmarks/GeneratedHydratorBenchmark/AllPrivateClassHydrationBench.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace GeneratedHydratorBenchmark;
+
+/**
+ * @BeforeMethods({"setUp"})
+ */
+class AllPrivateClassHydrationBench extends AbstractHydrationBench
+{
+    public function setUp()
+    {
+        $this->createHydrator(AllPrivateClass::class);
+        $this->createData();
+    }
+
+    /**
+     * @Revs(100)
+     * @Iterations(200)
+     */
+    public function benchConsume()
+    {
+        $object = new AllPrivateClass();
```

Comment:
Multiple hydration of the same object is not a problem: if the issue is bias of hydrating the same object, we can update an internal index and hydrate a series of objects instead.